### ISSUE TITLE
Cannot get form errors to display when submitting an empty form

### ISSUE THREAD

**Issue by aileen-r** [inkline/inkline#273]
**What is the expected and current behavior?**
I would expect that if a user were to submit an empty and untouched form, validation error messages would appear to explain why the form did not submit.

I would expect to be able to set all form fields to touched on submit to trigger validation with Inkline. However, the "submit" event from IForm does not get emitted if validation fails so I cannot hook into this event and force fields to "touched".

**Steps to reproduce**
I've put together a [minimal form](https://github.com/aileen-r/inkline-form-validation-no-errors-on-submit) with some comments in the [form component](https://github.com/aileen-r/inkline-form-validation-no-errors-on-submit/blob/main/src/components/InklineForm.vue).

**Please tell us about your environment:**
    - Version: >=3.0.8
    - Browser: all
    - Language: all

**Other information**
I tried the "update:modelValue" event which _is_ emitted on an empty submit, but the model hasn't changed so I can't find anything to identify that a submit has happened.

**Comment by Eternal-Rise** [comment]
Should it work by default? 

```vue
<script lang="ts" setup>
import { useForm } from '@inkline/inkline/composables';
import { validate } from '@inkline/inkline/validation';

const form = useForm({
    username: {
        validators: ['required']
    },
    password: {
        validators: ['required']
    }
});

const onSubmit = () => {
    console.log('SUBMIT', validate(form.value));
};
</script>

<template>
    <IForm v-model="form">
        <IFormGroup>
            <IFormLabel>Username</IFormLabel>
            <IInput name="username" placeholder="Type something.." />
            <IFormError for="username" />
        </IFormGroup>
        <IFormGroup>
            <IFormLabel>Password</IFormLabel>
            <IInput name="password" placeholder="Type something.." />
            <IFormError for="password" />
        </IFormGroup>

        <IFormGroup>
            <IButton type="button" @click="onSubmit">Submit</IButton>
        </IFormGroup>
    </IForm>
</template>
```

I tried this one and nothing changed. `form` is still untouched.

**Comment by alexgrozav** [comment]
Thank you for reporting the issue! Fix released in v4.4.0.

**Comment by alexgrozav** [comment]
@all-contributors add @aileen-r for bug report.
